ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -18828,7 +18828,16 @@ void HqlCppTranslator::buildTimerBase(BuildCtx & ctx, CHqlBoundExpr & boundTimer
     HqlExprArray registerArgs;
     registerArgs.append(*getSizetConstant(activityId));
     registerArgs.append(*createConstant(name));
-    OwnedHqlExpr call = bindFunctionCall(registerTimerId, registerArgs);
+    OwnedHqlExpr call;
+    if (statsOption == 0)
```

Comment:
clearer to

if (statsOption==ThorStatDefault)

Reply:
I agree, but making ThorStatDefault visible would require including a header and I am loathe to do so at this point. I did add a comment here, nothing what the zero actually means. I can, of course, include the header and use the enum value name instead if you really think it is worth it; let me know.